checkout/typescript/payment-methods/default/package.json

```diff
@@ -25,7 +25,7 @@
   "scripts": {
     "compile": "tsc",
     "prebuild": "webpack",
-    "build": "javy build/index.js -o build/index.wasm",
+    "build": "shopify script javy build/index.js -o build/index.wasm",
```

Comment:
We need to wait for a new release of the Shopify CLI to change this. There is currently a new release is in flight, but some times it takes the CLI team a couple of hours to finish the release because there are some manual steps. Either way, I have https://github.com/Shopify/scripts-apis-examples/pull/17 open which I can merge once thats done

Reply:
The javy command also needs to be run with an `-i` flag to specify the input file: `shopify script javy -i build/index.js -o build/index.wasm`. I'll open a PR in script apis now to update that. 

So in this PR you can either make those updates yourself or remove the changes to the build commands and I can merge that open PR when its ready. Up to you!

Reply:
Thanks for the callout. I'll remove the updates to this command and just keep the bump to version number